manifests/t/Tencent/TencentDocs/3.5.0/Tencent.TencentDocs.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created with komac v2.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Tencent.TencentDocs
+PackageVersion: 3.5.0
+InstallerType: nullsoft
+Scope: machine
+InstallerSwitches:
+  Upgrade: --updated
+UpgradeBehavior: install
+ReleaseDate: 2024-04-17
+Installers:
+- Architecture: x64
+  InstallerUrl: https://desktop.docs.qq.com/Installer/30001/3.5.0/TencentDocs-x86_64.exe
```

Comment:
I can't determine the direct link to this installer file at domain `dldir1v6.qq.com` :disappointed:

Reply:
Feel free to update the manifest later, if it's found.